pkg/components/exporter/exporter.go

```diff
@@ -37,40 +36,29 @@ import (
 )
 
 const (
-	prefix = "kepler-exporter-"
+	// prefix = "kepler-exporter-"
 
-	SCCName = prefix + "scc"
+	// SCCName = prefix + "scc"
 
-	ServiceAccountName   = prefix + "sa"
-	FQServiceAccountName = "system:serviceaccount:" + components.Namespace + ":" + ServiceAccountName
+	// ServiceAccountName   = prefix + "sa"
+	// FQServiceAccountName = "system:serviceaccount:" + k.Namespace() + ":" + ServiceAccountName
+	//
+	// ClusterRoleName        = prefix + "cr"
+	// ClusterRoleBindingName = prefix + "crb"
 
-	ClusterRoleName        = prefix + "cr"
-	ClusterRoleBindingName = prefix + "crb"
-
-	ConfigmapName = prefix + "cm"
-	DaemonSetName = prefix + "ds"
```

Comment:
Shouldn't we keep `ds` name constant as `kepler-exporter` ?

Reply:
what would happen if the 2 keplerinternals are deployed to the the same namespace with the daemonset name set as a const 💭 ?